### DISCUSSION TITLE
Netbox causes RST flood

### DISCUSSION THREAD

**Discussion by Zortoxon** [netbox-community/netbox#10966] [Q&A]
Hi all

Any good ideas why our netbox causes our sonicwalls to report RST floods against a single machine (my own) when accessing the website?

/Zortoxon

![MicrosoftTeams-image (1)](https://user-images.githubusercontent.com/89907389/202670773-f175065c-1480-4ac4-ad97-e4bcb23a9e74.png)

**Comment by kkthxbye-code** [comment]
Best guess would be a bug/misconfiguration in your firewall, your network or the webserver. My TCP fundementals are a little iffy, but I believe that the log indicates that your client is sending a very large amount of packets over a closed TCP session. Not sure how that would be possible, but netbox itself doesn't touch the TCP layer at all.

It seems very unlikely that your client would send 71k packages per second, so I'm guessing it's a bug. Try wireshark and see if the traffic is actually generated.
